### PR TITLE
sqls done

### PR DESCRIPTION
**Pull request by calin2110**

### CODE REVIEW

**Comment by alex-nedelcu** [comment]
<b>Pe branch-ul de pe care ai facut acest PR</b>, fa urmatoarele modificari:
- adauga `.idea` in fisierul `/backend/.gitignore`
- schimba parola pentru userul `postgres` din psql din '1234' in 'postgres' ca sa avem toti acelasi fisier `application.properties` (vezi pasul 4 de la Notes https://docs.google.com/document/d/1eunQSh_BJow_vKPZCmy6cGXW9TqgmTX7UKgIY8MTCMc/edit)
- dupa ce ai facut modificarile de mai sus: `git add .` + `git commit -m "Solved review issues"` + `git push`

File: backend/src/main/java/com/example/models/ActiveContract.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import java.sql.Date;
+
+@Entity
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "active_contracts")
+@IdClass(CompositeKey.class)
+// this should DEFINITELY not work
```
Comment (eduardpauliuc) [inline backend/src/main/java/com/example/models/ActiveContract.java:18]: 🥲

Comment (calin2110) [reply backend/src/main/java/com/example/models/ActiveContract.java:18]: for some whatever reason, stackoverflow says that it is working...?

Comment (calin2110) [reply backend/src/main/java/com/example/models/ActiveContract.java:18]: but it makes no sense to some extent

Comment (calin2110) [reply backend/src/main/java/com/example/models/ActiveContract.java:18]: https://stackoverflow.com/questions/71626588/modeling-entities-in-springboot-while-having-the-database-diagram-using-hibernat

Comment (calin2110) [reply backend/src/main/java/com/example/models/ActiveContract.java:18]: https://stackoverflow.com/questions/59695741/jpa-how-to-handle-the-entity-mapping-with-two-foreign-keys-and-those-two-forei

**Pull request merged**